bin/test.sh

```diff
@@ -7,21 +7,30 @@ set -euo pipefail
 STACK="${1}"
 BASE_IMAGE="heroku/${STACK/-/:}-build"
 OUTPUT_IMAGE="redis-test-${STACK}"
+REDIS_CI_TLS=${REDIS_CI_TLS:-}
 
 echo "Building buildpack on stack ${STACK}...with redis version ${REDIS_VERSION}"
 
 docker build \
+    --no-cache \
     --build-arg "BASE_IMAGE=${BASE_IMAGE}" \
     ${REDIS_VERSION:+--build-arg "REDIS_VERSION=${REDIS_VERSION}"} \
+    ${REDIS_CI_TLS:+--build-arg "REDIS_CI_TLS=${REDIS_CI_TLS}"} \
     -t "${OUTPUT_IMAGE}" \
     .
 
 echo "Checking redis-server presence and version..."
 
+if [ -n "$REDIS_CI_TLS" ]; then
```

Comment:
Is there a world where we default this to TLS (next major)? 

I'd be surprised if many devs finds this setting.

Reply:
Yes, I think so - didn't just want to break folks immediately.